airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams.py

```diff
@@ -280,8 +279,6 @@ class AdsInsights(FBMarketingIncrementalStream):
         "action_destination",
     ]
 
-    MAX_WAIT_TO_START = pendulum.duration(minutes=5)
-    MAX_WAIT_TO_FINISH = pendulum.duration(minutes=30)
     MAX_ASYNC_SLEEP = pendulum.duration(minutes=5)
```

Comment:
Maybe we should increase this number. 5 min to wait for a job feels too low

Reply:
It would be nice to have some logs for statistics to know exactly how often this happens, but from the logs that I have seen it almost never exceed this time.
Will increase this value though